code/game/objects/structures/signs/signs_misc.dm

```diff
@@ -12,3 +12,12 @@
 
 /obj/structure/sign/gym/mirrored/right
 	icon_state = "gymmirror-right"
+
+/obj/structure/sign/xenobio_guide
+	name = "\improper Slime genealogy sign"
+	sign_change_name = "Xenobiology guide"
+	desc = "A sign depicting the slime mutation map."
```

Comment:
I'd recommend being a little more specific, for the newer players.
`"A sign depicting the slime mutation map. By starting at the grey slime in the center, one can mutate into the variants branching outwards."`
The above line is very clunky but it gets what I mean across, I'm sure you could think up a better explanation line than I did lmao.

Reply:
Newer players can't do anything related to xenobiology without the wiki anyway, and they will see this mutation map there. This is a memo of that map, rather than a tutorial for newer players.

Reply:
I still think the clarification would be nice in some way, but I'm not a maintainer so I can't really make you do that